packages/react-native/ReactCommon/React-FabricComponents.podspec

```diff
@@ -92,7 +92,7 @@ Pod::Spec.new do |s|
     "react/renderer/components/view/platform/cxx",
     "react/renderer/imagemanager/platform/ios"
   ])
-  add_dependency(s, "ReactCodegen", :additional_framework_paths => ["build/generated/ios"])
+  # add_dependency(s, "React-RCTFBReactNativeSpec")
```

Comment:
Guessing this means React Fabric components doesn't have a dependency on the outputted codegened files? Why keep the comment though?